Engine/gfx/ali3dsw.cpp

```diff
@@ -376,6 +376,10 @@ bool ALSoftwareGraphicsDriver::Init(int virtualWidth, int virtualHeight, int rea
 
     virtualScreen = BitmapHelper::GetScreenBitmap();
 
+    // [ER] 2014-03-13
+    // Hide the system cursor via allegro
+    show_os_cursor(MOUSE_CURSOR_NONE);
```

Comment:
Edward Rudd's specific commit comment about this was "hide the stupid mouse cursor all the time on ags graphics startup". :)  However it does hide the mouse earlier while windows in OSX and are animating to fullscreen. (there may be a reason for it in Linux too)